output/csv/config.go

```diff
@@ -94,10 +104,7 @@ func ParseArg(arg string, logger *logrus.Logger) (Config, error) {
 		case "fileName":
 			c.FileName = null.StringFrom(r[1])
 		case "timeFormat":
-			c.TimeFormat = TimeFormat(r[1])
-			if !c.TimeFormat.IsValid() {
```

Comment:
I'd suggest checking if `c.TimeFormat.IsValid()`, and if it's not, write a warning log with a message that was initially an error.

Reply:
I do in https://github.com/grafana/k6/pull/2560/files#diff-1953fa5bc9d0d1b36a24a12f639d819ce02774065272f16e82a1162dd5fc9710R89-R92